SF iOS/SF iOS/GroupSelection/GroupCollectionViewController.m

```diff
@@ -95,6 +95,23 @@ - (CGFloat)collectionView:(UICollectionView *)collectionView layout:(UICollectio
     return 8.0;
 }
 
+- (BOOL)collectionView:(UICollectionView *)collectionView shouldHighlightItemAtIndexPath:(NSIndexPath *)indexPath {
+    UICollectionViewCell *cell = [collectionView cellForItemAtIndexPath:indexPath];
+    cell.layer.transform = CATransform3DMakeScale(.97, .97, 1);
+
+    const CATransform3D transformFromValue = cell.layer.transform;
+
+    [cell.layer addAnimation:AnimationWithKeyPath(@"transform", [NSValue valueWithCATransform3D:transformFromValue]) forKey:@"transform"];
+    return true;
+}
+
+static CABasicAnimation *AnimationWithKeyPath(NSString *keyPath, NSValue *fromValue) {
```

Comment:
Would be nice to move this to some sort of Animation Helper / Util file rather than living in the viewcontroller.

Reply:
This could either be a helper file and call the animation code manually, or it could be an extension or subclass of collectionview cell that calls the code on `setHighlighted:` and have all "card-like" cells in the app just subclass that

Reply:
I think the best play is to make a `BouncyCell: UICollectionViewCell` class and have both `FeedItemCell` and the cells used in this collectionview subclass `BouncyCell` which does a down / up animation on `setHighlighted:`

Reply:
Thanks Ethan.  Created new pull request, check it out.